Cork/App State.swift

```diff
@@ -9,11 +9,12 @@ import Foundation
 import AppKit
 import UserNotifications
 
+@MainActor
 class AppState: ObservableObject {
     @Published var navigationSelection: UUID?
     
     @Published var notificationEnabledInSystemSettings: Bool?
-    @Published var notificationStatus: UNNotificationSettings?
+    @Published var notificationAuthStatus: UNAuthorizationStatus = .notDetermined
```

Comment:
`UNNotificationSettings` is not sendable. The app doesn't use any properties on `UNNotificationSettings ` other than the authorization status enum though, which is sendable, so the simplest thing is to store it instead.

Reply:
Good idea